From 1320789ec61571ca3809c284cac277adec0b5d76 Mon Sep 17 00:00:00 2001
From: Xing Zheng <zhengxing@rock-chips.com>
Date: Thu, 16 Aug 2018 17:14:59 +0800
Subject: [PATCH] ASoC: rk3308_codec: increase ADC_STABLE_MS to 200ms

It looks that switching BIST mode to normal mode, we need
to increase the ADC_STABLE_MS delay (> ~150ms) to avoid
glitches.

Change-Id: Ied7b283cd4a2fa8c2702a655dab464a7f90f63c4
Signed-off-by: Xing Zheng <zhengxing@rock-chips.com>
---
 sound/soc/codecs/rk3308_codec.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/codecs/rk3308_codec.c b/sound/soc/codecs/rk3308_codec.c
index c3bc491bf947..1a78d20fecea 100644
--- a/sound/soc/codecs/rk3308_codec.c
+++ b/sound/soc/codecs/rk3308_codec.c
@@ -55,7 +55,7 @@
 
 #define ADC_GRP_SKIP_MAGIC		0x1001
 #define ADC_LR_GROUP_MAX		4
-#define ADC_STABLE_MS			20
+#define ADC_STABLE_MS			200
 #define DEBUG_POP_ALWAYS		0
 #define ENABLE_AGC			0
 #define HPDET_POLL_MS			2000
-- 
2.35.3

